src/outboundApi/handlers.js

```diff
@@ -40,7 +40,7 @@ const postTransfers = async (ctx) => {
         ctx.response.body = response;
     }
     catch(err) {
-        ctx.response.status = 500;
+        ctx.response.status = err.httpStatusCode || 500;
```

Comment:
I think you also need to add this change to the putTransfers handler - if we are in 3 or 2 stage mode, model might throw on missed deadlines also?